page-container/index-page/IndexPageContainer.tsx

```diff
@@ -74,39 +72,37 @@ const IndexPageContainer = () => {
   const classes = useStyle();
   return (
     <>
-      <Container disableGutters fixed>
+      <Box mx="10%">
```

Comment:
`<Box mx="10%">` seems still the best way in this case, better responsive styling and readable tag